test/patest_read_record.c

```diff
@@ -47,13 +47,14 @@
 #include <stdlib.h>
 #include "portaudio.h"
 
-/* #define SAMPLE_RATE  (17932) // Test failure to open with this value. */
-#define SAMPLE_RATE  (44100)
-#define FRAMES_PER_BUFFER (1024)
-#define NUM_SECONDS     (5)
-#define NUM_CHANNELS    (2)
-/* #define DITHER_FLAG     (paDitherOff)  */
-#define DITHER_FLAG     (0) /**/
+/* #define SAMPLE_RATE        (17932) // Test failure to open with this value. */
+#define SAMPLE_RATE        (44100)
+#define FRAMES_PER_BUFFER  (1024)
+#define NUM_SECONDS        (5)
+#define NUM_CHANNELS       (2)
+/* #define DITHER_FLAG        (paDitherOff)  */
+#define DITHER_FLAG        (0) /**/
+#define USE_LOOPBACK_INPUT (0)
```

Comment:
Did you mean to include the changes to this file? If you want a test for wasapi loopback read_record, please move it to a wasapi-specific test like you did for read_write_wire_wasapi.

Reply:
Sorry, I did not notice. Reverted `patest_read_record` to original state, introduced `patest_read_record_wasapi` as separate file.